.changeset/sour-spiders-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-bitbucket-server': minor
```

Comment:
```suggestion
'@backstage/plugin-catalog-backend-module-bitbucket-server': patch
```

Can you change this to a patch? Minor version bumps when the plugin is on a zeroth version are considered patched. 🙏